docs/releases/unreleased.md

```diff
@@ -0,0 +1,20 @@
+# OPA Core Module version XXX
+
+SIGHUP team maintains this module updated and tested. That is the main reason why we worked on this new release.
+With the Kubernetes 1.21 release, it became the perfect time to start testing this module against this Kubernetes
+release.
+
+Continue reading the [Changelog](#changelog) to discover them:
+
+## Changelog
+
+- Add a Constraint Template to protect namespaces for being deleted.
+
+## Upgrade path
+
+To upgrade this core module from `v1.3.1`, you need to download this new version, then apply the
+`kustomize` project. No further action is required.
+
```

Comment:
We should say that there's a breaking change: now we are watching the DELETE events also. If people are using their own rules they must be aware of this change, they probably might have to do the same fix that we had to.